openmetadata-ui/src/main/resources/ui/src/components/Database/DatabaseSchema/DatabaseSchemaTable/DatabaseSchemaTable.tsx

```diff
@@ -152,18 +158,10 @@ export const DatabaseSchemaTable = ({
         query: '',
         pageNumber,
         pageSize: PAGE_SIZE,
-        queryFilter: getTermQuery(
-          { 'database.fullyQualifiedName': decodedDatabaseFQN },
-          'must',
-          undefined,
+        queryFilter: buildSchemaQueryFilter(
```

Comment:
what's the need for having the `buildSchemaQueryFilter` if we are having `getTermQuery` ?